janusgraph-test/src/main/java/org/janusgraph/diskstorage/indexing/IndexProviderTest.java

```diff
@@ -1080,10 +1089,10 @@ public void testScroll() throws BackendException {
     public void testPersistantIndexData() throws BackendException {
         final String store = "vertex";
 
-        final Multimap<String, Object> initialDoc = HashMultimap.create();
+        final Multimap<String, Object> initialDoc = ArrayListMultimap.create();
         initialDoc.put(DATE, Instant.ofEpochSecond(1));
-        double latitude = random.nextDouble() * 180 - 90;
-        double longitude = random.nextDouble() * 360 - 180;
+        double latitude = 30;
+        double longitude = 60;
```

Comment:
Would you be able also to add this flaky test fix into `0.3` branch?

Reply:
Ok, https://github.com/JanusGraph/janusgraph/pull/1696